kamon-akka-http/src/main/scala/kamon/akka/http/AkkaHttp.scala

```diff
@@ -61,6 +61,15 @@ object AkkaHttp {
     }
   }
 
+  val addHttpStatusCodeAsMetricTag: Boolean = {
+    val statusAsMetricTagConfigName = "kamon.akka-http.http-status-code-as-metric-tag"
+    if (Kamon.config.hasPath(statusAsMetricTagConfigName)) {
```

Comment:
We usually let things break if a configuration key is not found since that is definitely some exceptional issue, there should always be some value for this key, either provided by the user or from the `reference.conf` file so, not so much need to be double safe here :)